nio/src/main/scala/zio/nio/package.scala

```diff
@@ -64,10 +62,11 @@ package object nio {
      */
     def useForked[R2 <: R, E2 >: E, B](
       f: A => ZIO[R2, E2, B]
-    )(implicit trace: ZTraceElement): ZIO[R2, E, Fiber[E2, B]] = ReleaseMap.make.flatMap { releaseMap =>
-      managed.zio.flatMap { case (finalizer, a: A) =>
-        f(a).onExit(finalizer).fork
+    )(implicit trace: ZTraceElement): ZIO[R2, E, Fiber[E2, B]] =
+      ReleaseMap.make.flatMap { releaseMap =>
+        managed.zio.flatMap { case (finalizer, a: A) =>
+          f(a).onExit(finalizer).fork
+        }
```

Comment:
Here's the other compiler error.

Reply:
Might get quicker responses if you include the errors themselves, in addition to the locations 🙂 
Won't require other folks to checkout the branch and compile it.
(Disclaimer: _I_ probably won't have any answers, but it could assist those that do)

Reply:
I thought about that and hoped it would be simple enough to include informative compiler errors. Unfortunately, in these cases, the compiler errors say "value _x_ is not a member of _y_", which boils down to differences between the addition of scopes between ZIO 2.0.0-RC2 and ZIO 2.0.0-RC3.

For this error, here is the closest Rosetta Stone I could find as a general pattern for refactoring this small section of code:

https://github.com/zio/zio/pull/6420/files#diff-6c2408471a67648e396de3be291162f2dd677d642b3f712d63ad5fa8308f3629L510

However, I just wasn't sure precisely what I needed to do here.